README.md

```diff
@@ -1 +1,14 @@
-# Hello-Microverse
\ No newline at end of file
+Hello Microverse is a simple project. It is the first project created as part of microverse curriculam.
+
+Tech Stack
+Client 
+HTML/CSS
+
+Authors
+
+Author1
+
+GitHub: @iamsjunaid
+Twitter: @junaidahmedsd
+LinkedIn: https://www.linkedin.com/in/junaid-syed-058b2779
+
```

Comment:
- [ ] Kindly 😊 use [the Microverse Readme template](https://github.com/microverseinc/readme-template) and make sure all the main sections present in yours and placeholders are replaced with your project and personal data. The only optional section is a FAQ. Please remove the template logo on the top, if you have one, you can replace it. This is crucial to have a professional and instructive repository. 👌